src/dss/main.py

```diff
@@ -30,5 +31,40 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+CREATE_NOTEBOOK_IMAGE_HELP = """
```

Comment:
We are going to change this soon (when we have our rocks) could we have either a tracking issue or a comment that says these images are not the final ones that will be printed in the help?

Reply:
good call - captured as #57